ci/test-abi/google_cloud_cpp_common.expected.abi.dump

```diff
@@ -0,0 +1,564 @@
+$VAR1 = {
+          'ABI_DUMPER_VERSION' => '1.1',
+          'ABI_DUMP_VERSION' => '3.5',
+          'Arch' => 'x86_64',
+          'Compiler' => 'GNU C++11 7.3.0 -mtune=generic -march=x86-64 -g -std=gnu++11 -fPIC -fstack-protector-strong',
+          'Headers' => {
+                         'build_info.h' => 1,
+                         'setenv.h' => 1,
+                         'throw_delegate.h' => 1
+                       },
+          'Language' => 'C++',
+          'LibraryName' => 'libgoogle_cloud_cpp_common.so',
+          'LibraryVersion' => '0.1.0',
+          'MissedOffsets' => '1',
+          'MissedRegs' => '1',
+          'NameSpaces' => {
+                            '__gnu_cxx' => 1,
+                            'google::cloud::v0::internal' => 1,
+                            'std' => 1,
+                            'std::__cxx11' => 1,
+                            'std::__exception_ptr' => 1
+                          },
+          'Needed' => {
+                        'libc.so.6' => 1,
+                        'libgcc_s.so.1' => 1,
+                        'libstdc++.so.6' => 1
+                      },
+          'PublicABI' => '1',
+          'Sources' => {
+                         'build_info.cc' => 1,
+                         'setenv.cc' => 1,
+                         'throw_delegate.cc' => 1
+                       },
+          'SymbolInfo' => {
+                            '16246' => {
+                                         'Data' => 1,
+                                         'Header' => 'build_info.h',
+                                         'Line' => '25',
+                                         'MnglName' => '_ZN6google5cloud2v08internal8COMPILERE',
+                                         'NameSpace' => 'google::cloud::v0::internal',
+                                         'Return' => '16316',
+                                         'ShortName' => 'COMPILER',
+                                         'Source' => 'build_info.cc',
+                                         'SourceLine' => '22'
```

Comment:
Almost certainly line numbers will change when source code changes...

Reply:
Yes.  The line numbers would change, but that does not affect the compatibility report.  For example, I made a local change to remove a function, which produced 
[compat_report.html.gz](https://github.com/GoogleCloudPlatform/google-cloud-cpp/files/2078024/compat_report.html.gz).  The line number changes had no effect, but the removed function is clearly identified.

I expect that we will commit new versions of this file only when the reports fail, which would  create a massive diff, but who cares?